src/Fmt.mli

```diff
@@ -58,6 +58,11 @@ val list_pn : 'a list -> (?prev:'a -> 'a -> ?next:'a -> t) -> t
 (** Format a list using provided function for the elements, which is passed
     the previous and next elements, if any. *)
 
+val list_sep :
+  'a list -> (Format.formatter -> unit -> unit)
+  -> ('a -> Format.formatter -> unit) -> Format.formatter -> unit
+(** Format a list using the first function for the separators between elements *)
```

Comment:
Nit: the other docstrings end with a period, and then it will probably need to be re-wrapped.